test/wallet-test.js

```diff
@@ -18,7 +18,7 @@ const KeyRing = require('../lib/primitives/keyring');
 const Input = require('../lib/primitives/input');
 const Outpoint = require('../lib/primitives/outpoint');
 const Script = require('../lib/script/script');
-const PrivateKey = require('../lib/hd/private.js');
+const HDPrivateKey = require('../lib/hd/private');
```

Comment:
lol - this `.js` was leftover from an earlier port https://github.com/handshake-org/hsd/pull/97 I guess I opened the PR on hsd before it was finished on bcoin!